src/infrastructure/middleware/authorization/whiteList/string.compare.ts

```diff
@@ -0,0 +1,21 @@
+import { EntriesUtils } from "./entriesUtils";
+import { ICompare } from "./iCompare";
+import { PathEvaluation } from "./pathEvaluation";
+
+export class StringCompare implements ICompare {
```

Comment:
What do you think about move this feature to the StringUtil class?

Reply:
The idea here is to have the list item in memory ready to perform the comparison, so at the beginning of the microservice, the whitelist items are parsed once, the objective is to have the minimum impact on performance, since it is a loop that will be performed on all requests.
If it is moved to StringUtil, you run the risk of parsing it multiple times.

What is your recommendation?

Reply:
I still think that this class is a domain and can be a utility that someone can reuse in the future. 
It's logic, normal logic, you can use it in any module into de solution, so the performance is not going to be affected in any way because is only computed code.

Reply:
Got it bro, I created the startWithCollection method in StringUtil